### PR TITLE
setup automation for cc uar quiz

### PR DESCRIPTION
**Pull request by JackySun9**
1. test uar live page for CC

### CODE REVIEW

File: features/cc/quiz.dynamic.spec.js [hunk 1]
```diff
@@ -0,0 +1,194 @@
+module.exports = {
+  name: 'Quiz Recommender Blocks',
+  features: [
+    {
+      tcid: '10',
```
Comment (amauch-adobe) [inline features/cc/quiz.dynamic.spec.js:5]: not that it matters too much, but why are the tcid numbers starting so high and then jumping to the 20's and 30's?  Just wondering if there was a reason for it or if it should start at the normal 0 and forward numbers.

File: features/cc/quiz.spec.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
+module.exports = {
+  name: 'Quiz Recommender Blocks',
+  features: [
+    {
+      tcid: '000',
```
Comment (amauch-adobe) [inline features/cc/quiz.spec.js:5]: Is there a reason for the tcid being '000' and not just '0' and then the rest of the tcids in this file following a normal increment?

File: tests/cc/quiz.dynamic.test.js
Comment (amauch-adobe) [inline tests/cc/quiz.dynamic.test.js:1]: Lint error should be resolved.

File: tests/uar/quiz.screenshots.test.js
Comment (amauch-adobe) [inline tests/uar/quiz.screenshots.test.js:1]: Lint error should be resolved.

**Review by amauch-adobe — CHANGES_REQUESTED** [review]
See my comments and questions on the spec.js files.  Also, it would be best to have the Linter errors resolved before being merged.

File: features/cc/quiz.dynamic.spec.js (hunk 1 above)
Comment (JackySun9) [reply features/cc/quiz.dynamic.spec.js:5]: I use id to category them

File: features/cc/quiz.spec.js (hunk 2 above)
Comment (JackySun9) [reply features/cc/quiz.spec.js]: ok, I can update this

**Review by amauch-adobe — APPROVED** [review]
Approved.

**Pull request merged**